idea_town/frontend/static-src/app/templates/experiment-page.js

```diff
@@ -22,55 +22,59 @@ export default `
       <div class="details-content">
         <div class="details-overview">
           <img src="{{thumbnail}}" width="260">
-          <section>
+          {{#model.measurements}}
+          <section class="measurement">
             <h3>Measurements</h3>
-            <p class="disclaimer">All data is collected anonymously and used only to help us improve this test. <a href="">Learn more.</a></p>
+            <p class="disclaimer">All data is collected anonymously and used only to help us improve this test.</p>
+            {{{model.measurements}}}
+            <!--
             <ul class="measurement">
```

Comment:
It looks like the css was modified to apply styles to this via `.measurement ul`, so you probably want to remove the `measurement` class here